remote-query/remote-query-server/src/test/java/org/infinispan/query/remote/impl/ProtobufIndexedFieldProviderTest.java

```diff
@@ -0,0 +1,68 @@
+package org.infinispan.query.remote.impl;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.Index;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.protostream.FileDescriptorSource;
+import org.infinispan.protostream.SerializationContext;
+import org.infinispan.test.SingleCacheManagerTest;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.infinispan.transaction.TransactionMode;
+import org.testng.annotations.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author anistor@redhat.com
```

Comment:
@slaskawi We need more test comments, not less :)
@anistor How about a short description?